package.json

```diff
@@ -47,14 +47,16 @@
     ]
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.0.0"
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
```

Comment:
I didn't intend for react to be bundled. Should this be in peer dependencies instead?

Reply:
Hm, I see. I'm not really an expert on these things, but I guess turning them into peer dependencies would be the best solution as you do need the packages to be present after all.